addons/website/static/src/components/dialog/page_properties.js

```diff
@@ -155,108 +154,122 @@ DuplicatePageDialog.template = xml`
 </WebsiteDialog>
 `;
 DuplicatePageDialog.props = {
-    onClose: Function,
-    close: Function,
+    onDuplicate: {type: Function, optional: true},
     pageId: Number,
+    close: Function,
+};
+
+export const pagePropertiesProps = {
+    title: _t('Page Properties'),
+    resModel: 'website.page',
+    context: {
+        form_view_ref: 'website.website_page_properties_view_form',
+    },
 };
 
-export class PagePropertiesDialogWrapper extends Component {
+export class PagePropertiesDialog extends FormViewDialog {
     setup() {
-        try {
-            this.websiteService = useService('website');
-            this.dialogService = useService('dialog');
-            this.orm = useService('orm');
-        } catch {
-            // Use services in legacy environment.
-            this.websiteService = useWowlService('website');
-            this.dialogService = useWowlService('dialog');
-            this.orm = useWowlService('orm');
-        }
+        super.setup();
+        this.dialog = useService('dialog');
+        this.website = useService('website');
```

Comment:
Mmh... I see that half the time we used "Service" in the name and half the time we did not :grimacing: I see that the framework JS team seems to use the version without the service word indeed. If you confirm it, could you make a dedicated commit which renames those variables in all of our components first? So that we avoid diff in these commits and so that we enforce the convention everywhere once and for all :+1: (either in another PR or this one, as you prefer. If there is many things to review after my review, maybe do that in another PR first).

Reply:
> I see that the framework JS team seems to use the version without the service word indeed

I actually did this because they use it without "Service" most of the time, but I don't think there is a convention about this (on [Odoo frontend DOC](https://www.odoo.com/documentation/master/developer/reference/frontend/services.html?highlight=useservice) I found both (with & without "Service"), so I'm not sure about this...?

Reply:
Mmh, they seem to do both in the doc indeed... ok, let's not waste time and keep your diff about that in this commit. I guess we'll enforce a convention over time...